src/routes/(app)/cart/+page.svelte

```diff
@@ -346,7 +353,17 @@
 				</div>
 			{/if}
 			<div class="flex justify-end">
-				<a href="/checkout" class="btn body-cta body-mainCTA">{t('cart.cta.checkout')}</a>
+				{#if !physicalCartCanBeOrdered}
+					<p class="text-red-500 font-light">{t('cart.minimumPhysicalAmountText')}</p>
+				{/if}
+			</div>
+			<div class="flex justify-end">
+				<a
+					href="/checkout"
+					class="btn body-cta body-mainCTA {!physicalCartCanBeOrdered ? 'opacity-50' : ''}"
+					style="pointer-events: {!physicalCartCanBeOrdered ? 'none' : ''};"
+					>{t('cart.cta.checkout')}</a
+				>
```

Comment:
make it a `button` with `disabled` attribute instead in that case, instead of using `pointer-events` and `opacity`

By the way here too: https://github.com/be-BOP-io-SA/be-BOP/blob/2c170ca9bb7129859995cd39ac179bf9fa2c6182/src/lib/components/ProductWidget/ProductWidgetPOS.svelte#L58-L61

That way it benefits from being more correct for accessibility purposes and from the theming too